plugins/com.google.cloud.tools.eclipse.appengine.libraries/src/com/google/cloud/tools/eclipse/appengine/libraries/model/MavenCoordinates.java

```diff
@@ -193,6 +201,22 @@ public Builder setClassifier(String classifier) {
       return this;
     }
     
+  }
+
+  /**
+   * Check Maven Central to find the latest release version of this artifact.
+   * This check is made at most once. Subsequent checks are no-ops.
+   */
+  public void updateVersion() {
```

Comment:
`MavenCoordinates` was a lightweight, strictly immutable class, so I don't like adding an update method that much, in addition to the fact that his method makes a long-running network call. I prefer leaving this class as a stateless constant representation. To update a version, you'd want to utilize `.toBuilder()`, which was intended for updating things. That said, if you add a new field, you also need to modify `.toBuilder()`.